README.rst

```diff
@@ -34,6 +34,16 @@ For a list of commands and options, check the help output with:
     patrol --help
 
 
+Organization
```

Comment:
this list is getting a little big, maybe it's better to just leave the `patrol --help` ahead.

Reply:
Do you want me to put the `patrol --help` output too?

Reply:
No, exacly what you just did :)
merging